pyquil/operator_estimation.py

```diff
@@ -746,10 +762,24 @@ def measure_observables(qc: QuantumComputer, tomo_experiment: TomographyExperime
         to True is much faster but there is a ~1% error per qubit in the reset operation.
         Thermal noise from "traditional" reset is not routinely characterized but is of the same
         order.
+    :param symmetrize: Whether to symmetrize the readout errors, i.e. set p(0|1) = p(1|0). For
+        uncorrelated readout errors, this can be achieved by randomly selecting between the
+        POVMs {X.D1.X, X.D0.X} and {D0, D1} (where both D0 and D1 are diagonal). However, here
+        we use exhaustive symmetrization and loop through all possible 2^n POVMs {X/I . POVM . X/I}^n,
+        and obtain symmetrization more generally, i.e. set p(00|00) = p(01|01) = .. = p(11|11), as well
+        as p(00|01) = p(01|00) etc.
+    :param calibrate_readout: Whether to calibrate the readout results by normalizing against the
+        operator's expectation value in its +1 eigenstate. The preceding symmetrization and this step
+        together yield a more accurate estimation of the observable.
     """
+    # calibration readout only works with symmetrization turned on
```

Comment:
enforce this by raising exceptions if the user specifies an invalid configuration. Do not silently override the user

Reply:
Updated.